README.adoc

```diff
@@ -174,24 +228,101 @@ Same as in the `ne4j` repo.
 Suggestions for improvement::
 
 Give a possible solution for the provided example query.
+====
```

Comment:
Github would not let me mark that line, but there is a typo in Neo4j on line 226 🙈